modules/node-embeddings/pages/weisfeiler-lehman.adoc

```diff
@@ -0,0 +1,178 @@
+= Weisfeiler-Lehman
+:description: Overview pf the Weisfeiler-Lehman for detecting isomorphic graphs.
+:fn-iso-article: footnote:[David Bieber, The Weisfeiler-Lehman Isomorphism Test, https://davidbieber.com/post/2019-05-10-weisfeiler-lehman-isomorphism-test/]
+
+This algorithm identifies isomorphic subgraphs in a graph, and labels the corresponding vertices isomorphic graphs.
+
+Two graphs are considered isomorphic if there is a mapping between the nodes of the graphs that preserves node adjacencies.
+That is, a pair of nodes may be connected by an edge in the first graph if and only if the corresponding pair of nodes in the second graph is also connected by an edge in the same way.
+An example of two isomorphic graphs is shown here{fn-iso-article}:
+
+.Graph 1 and Graph 2 are isomorphic. The correspondence between nodes is illustrated by the node colors and numbers.
+image::graph-isomorphism.png[]
+
+This algorithm gives every vertex in the graph a label.
```

Comment:
Do you know how the labels are generated and if there's a maximum number of labels?